ParseLoginUI/build.gradle

```diff
@@ -1,19 +1,19 @@
 apply plugin: 'android-library'
 
 dependencies {
-    compile 'com.parse.bolts:bolts-android:1.2.0'
-    compile 'com.android.support:support-v4:22.0.0'
+    compile rootProject.ext.bolts
+    compile rootProject.ext.androidSupport
 
-    provided 'com.facebook.android:facebook-android-sdk:4.0.1'
+    provided rootProject.ext.facebookSDK
     // This assumes that your app's project has a compile dependency on the Parse SDK JAR files.
     // Your project's build.gradle should say:
     //
-    //     compile files('YOUR_PROJECT_LIBS_PATH/Parse-1.9.4.jar')
-    //     compile files('YOUR_PROJECT_LIBS_PATH/ParseFacebookUtilsV4-1.9.4.jar')
+    //     compile 'com.parse:parse-android:1.10.0'
```

Comment:
The Parse SDK was not in Maven before.  We had to do this because people usually already have a Parse SDK JAR (of possibly different version) in their top-level project.  If another Parse SDK JAR was also in ParseUI-Android folder, they would get confused when they copy the library folder into their app and get the "multiple dex files" error. 

Anyways, the previous use case is no longer applicable now that we have Parse SDK in Maven :)  So please ignore the previous setup.